randovania/server/multiplayer/session_admin.py

```diff
@@ -583,7 +594,7 @@ def _create_patcher_file(sa: ServerApp, session: MultiplayerSession, world_uid:
 def admin_player(sa: ServerApp, session_id: int, user_id: int, action: str, *args):
     monitoring.set_tag("action", action)
 
-    verify_has_admin(sa, session_id, user_id)
+    verify_has_admin(sa, session_id, sa.get_current_user().id)
```

Comment:
This needs a test

Reply:
On second thought, I'm not quite sure if I want to have this change
It was used so that on "everyone can claim worlds", that a user can unclaim a world from other people. But we probably shouldn't allow that.